src/app/components/RuntimeEvents/RuntimeEventDetails.tsx

```diff
@@ -215,8 +215,10 @@ const RuntimeEventDetailsInner: FC<{
               fontWeight={400}
             />
             <br />
-            {t('runtimeEvent.fields.emittingContract')}:{' '}
-            <AccountLink scope={scope} alwaysTrim address={emittingEthAddress || emittingOasisAddress} />
+            <Box sx={{ display: 'inline-flex', alignItems: 'center' }}>
+              {t('runtimeEvent.fields.emittingContract')}:{' '}
+              <AccountLink scope={scope} alwaysTrim address={emittingEthAddress || emittingOasisAddress} />
```

Comment:
Address is still not aligned properly, because of highlighting padding.

Reply:
You are correct, this is not the right place to fix this.